communitygosu.properties

```diff
@@ -0,0 +1,32 @@
+category=Languages
+description=Code Analyzer for Gosu
+homepageUrl=https://github.com/FRI-DAY/sonar-gosu-plugin
+archivedVersions=1.0.0,1.1.0,1.2.0
+publicVersions=1.2.1
+
+defaults.mavenGroupId=de.friday
+defaults.mavenArtifactId=sonar-communitygosu-plugin
+
+1.2.1.description=Initial Marketplace entry
+1.2.1.sqVersions=[8.9,LATEST]
+1.2.1.date=2023-04-13
+1.2.1.changelogUrl=https://github.com/FRI-DAY/sonar-gosu-plugin/releases/tag/v1.2.1
+1.2.1.downloadUrl=https://github.com/FRI-DAY/sonar-gosu-plugin/releases/download/v1.2.1/sonar-communitygosu-plugin-1.2.1.jar
+
+1.2.0.description=Plugin key change
```

Comment:
Absolutely no reason to list archived versions on entry to the Marketplace. The point of archiving a version is to guide users to upgrade. You shouldn't have anyone on these old versions.

Reply:
Done. :-)